.ci/workspace-startup/workspace-startup-test.groovy

```diff
@@ -0,0 +1,128 @@
+                    export START_HARD_FAILURE_TIMEOUT="${START_HARD_FAILURE_TIMEOUT}"
+                    export STOP_SOFT_FAILURE_TIMEOUT="${STOP_SOFT_FAILURE_TIMEOUT}"
+                    export STOP_HARD_FAILURE_TIMEOUT="${STOP_HARD_FAILURE_TIMEOUT}"
+                    locust -f "../${RELATIVE_PATH}/osioperf.py" --no-web -c `echo -e "$USER_TOKENS" | wc -l` -r 1 --only-summary --csv="$EXPORT_FILE"
+                    """
+                }
+            }
+        }
+        stage ("Generating zabbix report") {
+            when {
+                expression { currentBuild.currentResult == "SUCCESS" }
+            }
+            steps {
+                script {
+                    def long DATETIME_TAG = System.currentTimeMillis() / 1000L
+                    def test_log_file = readFile("${LOG_DIR}/${EXPORT_FILE}_requests.csv")
+                    def BASE_DIR = pwd()
+                    def lines = test_log_file.split("\n")
+                    sh "touch ${ZABBIX_FILE}"
+                    for (line in lines) {
+                        def elements = line.split(",")
+                        def method = elements[0].replace("\"","")
+                        if (method.equals("Method") || method.equals("None")) {
+                            continue
+                        }
+                        def name_host_metric = elements[1].replace("\"","").split("_")
+                        def name = name_host_metric[0]
+                        if (name.equals("getWorkspaces") | name.equals("getWorkspaceStatus")) {
+                            continue
+                        }
+                        def host = name_host_metric[1]
+                        def int average = elements[5]
+                        def output_basestring = "qa-".concat(host).concat(" ").concat("che-start-workspace.").concat(method).concat(".").concat(name)
+                        def output = output_basestring.concat(" ")
+                                     .concat(String.valueOf(DATETIME_TAG)).concat(" ")
+                                     .concat(String.valueOf(average))
+                        silent_sh "echo $output >> ${ZABBIX_FILE}"
+                    }
+                }
+            }
+        }
+        stage ("Reporting to zabbix") {
+            when {
+                expression { currentBuild.currentResult == "SUCCESS" }
+            }
+            steps {
+                silent_sh "zabbix_sender -vv -i ${ZABBIX_FILE} -T -z ${ZABBIX_SERVER} -p ${ZABBIX_PORT}"
+            }
+        }
+    }
+    post("Cleanup") {
+        always {
+            echo "Current build status: ${currentBuild.currentResult}"
+            deleteDir()
+        }
+        aborted {
+            echo "Jenkins job timed out."
+        }
+        failure {
+            echo "Job failed, resetting environment for all users."
+            // mail to: team@example.com, subject: 'The Pipeline failed :('
```

Comment:
Why is the mail not send?

Reply:
Issue created, OK for me to solve it in other PR.

Reply:
Created a separate issue for implementing mails https://github.com/redhat-developer/rh-che/issues/1597